migrating_to_2.0/commands.rst

```diff
@@ -115,24 +136,36 @@ There are some examples:
     conan install . -s zlib*:arch=x86 -o zlib*:shared=True
     conan install . -s zlib/1.2.11@user/channel:arch=x86 -o zlib/1.2.11:shared=True
 
-
+Commands with have been removed
+-------------------------------
 
 Removed "conan package"
------------------------
+^^^^^^^^^^^^^^^^^^^^^^^
 
 The ``conan package`` command has been removed. If you are developing a recipe and want to test that the package method
 is correct, we recommend using the ``conan export-pkg .`` instead and exploring the package folder in the cache to check
 if everything is ok.
 
 
 Removed "conan copy"
---------------------
+^^^^^^^^^^^^^^^^^^^^
 
 Do not use the ``conan copy`` command to change user/channel. Packages will be immutable,
 and this command will disappear in 2.0. Package promotions are generally done on the
 server-side, copying packages from one server repository to another repository.
 
 
+Removed "conan user"
+^^^^^^^^^^^^^^^^^^^^
+
+This has been replaced with :ref:<conan_v2_remote_login>
+
+Removed "conan config set"
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+we are no longer implementing file-editing commands in 2.0. A bit overkill `conan config set` to edit one file. Which should very rarely happen,
+the file is updated with `conan config install`.
```

Comment:
you can pass config commands with `-c` flag, it could be a reasonable replacement for most of the use-cases

Reply:
```suggestion
the file is updated with `conan config install`. Alternatively, you can use the command line and profiles to pass these values.
```